jquery-truncate-html/jquery-truncate-html.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for jQuery-truncate-html.js 
+// Project: https://github.com/kbwood/timeentry
+// Definitions by: Abraão Alves <https://github.com/abraaoalves>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../jquery/jquery.d.ts"/>
+
+interface ITruncateOptions{
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files
> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.